package.json

```diff
@@ -1,54 +1,54 @@
 {
   "name": "freesound",
-  "version": "1.0.0",
-  "description": "Static files (JS/CSS) for freesound",
+  "version": "0.1.0",
+  "description": "Freesound, the cool stuff",
   "main": "index.js",
   "scripts": {
-    "dev": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --progress --colors",
-    "deploy": "NODE_ENV=production ./node_modules/webpack/bin/webpack.js -p",
-    "eslint": "./node_modules/eslint/bin/eslint.js src/**/**/**/** --ignore-pattern=*.json"
+    "start": "webpack-dev-server --config freesound/static/webpack/webpack.config.dev.js",
+    "build": "webpack --config freesound/static/webpack/webpack.config.js",
+    "build:prod": "webpack --config freesound/static/webpack/webpack.config.prod.js",
+    "prettier": "prettier freesound/static/src/**/**.js --write",
+    "eslint": "eslint freesound/static/src",
+    "styleguide": "styleguidist server --config freesound/static/styleguide.config.js",
+    "styleguide:build": "styleguidist build --config freesound/static/styleguide.config.js"
   },
-  "author": "Andres Ferraro",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/MTG/freesound.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/MTG/freesound/issues"
+  },
+  "homepage": "https://github.com/MTG/freesound#readme",
   "devDependencies": {
-    "deep-freeze": "0.0.1",
-    "doctrine": "^1.2.2",
-    "eslint": "^2.12.0",
-    "eslint-config-airbnb": "^9.0.1",
-    "eslint-config-react": "^1.1.1",
-    "eslint-plugin-import": "^1.8.1",
-    "eslint-plugin-jsx-a11y": "^1.4.2",
-    "eslint-plugin-react": "^5.1.1",
-    "jasmine-core": "^2.4.1",
-    "nock": "^8.0.0",
-    "phantomjs-prebuilt": "^2.1.7",
-    "rimraf": "^2.5.2",
-    "webpack-dev-server": "^1.14.1"
+    "autoprefixer": "^7.1.6",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-env": "^1.6.1",
+    "css-loader": "^0.28.7",
+    "eslint": "^4.12.1",
+    "eslint-config-airbnb-base": "^12.1.0",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-import": "^2.8.0",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "html-webpack-plugin": "^2.30.1",
+    "node-sass": "^4.7.2",
+    "postcss-flexbugs-fixes": "^3.2.0",
+    "postcss-loader": "^2.0.9",
+    "prettier": "^1.8.2",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
```

Comment:
`react` and `react-dom` have been added only to have the `styleguide` correctly running